<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUIFile</class>
 <widget class="QMainWindow" name="SettingsUIFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">b</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;</string>
    </property>
    <property name="title">
     <string>Alerts</string>
    </property>
    <widget class="QLineEdit" name="really_really_light_weigth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really really heavy weight:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really heavy weight:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_light_weigth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really really light weight:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really light weight:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_really_heavy_weigth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_heavy_weigth">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Time offset seconds:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="secs_time_offset">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_really_tall_height">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_really_short_height">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really short height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really really tall height:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_short_height">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really really short height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really tall height:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_tall_height">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="really_long_name_length">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>280</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Short name alert length:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="short_name_length">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Really long name length:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>351</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 3px solid black;</string>
    </property>
    <property name="title">
     <string>Errors</string>
    </property>
    <widget class="QLineEdit" name="max_mo_production_time">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>260</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_name_length">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>300</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Maximum molybdeen production time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Maximum technetium production time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Mininum weight:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Minimum technetium production time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_height">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_te_radioactivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Mininum name length:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_te_production_time">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Minimum molybdeen radioactivity:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_weight">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Min height length:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_te_production_time">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Maximum dose radioactivity:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_mo_radioactivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Minimum molybdeen radioactivity:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_mo_radioactivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Minimum molybdeen production time:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_dose_radioactivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_te_radioactivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_mo_production_time">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>240</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Minimum dose radioactivity:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Maximum technetium radioactivity:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="min_dose_radioactivity">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid gray;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Minimum technetium radioactivity:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="defaultButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset to default</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>really_really_light_weigth</tabstop>
  <tabstop>really_light_weigth</tabstop>
  <tabstop>really_heavy_weigth</tabstop>
  <tabstop>really_really_heavy_weigth</tabstop>
  <tabstop>secs_time_offset</tabstop>
  <tabstop>really_really_short_height</tabstop>
  <tabstop>really_short_height</tabstop>
  <tabstop>really_tall_height</tabstop>
  <tabstop>really_really_tall_height</tabstop>
  <tabstop>short_name_length</tabstop>
  <tabstop>really_long_name_length</tabstop>
  <tabstop>min_dose_radioactivity</tabstop>
  <tabstop>max_dose_radioactivity</tabstop>
  <tabstop>min_te_radioactivity</tabstop>
  <tabstop>max_te_radioactivity</tabstop>
  <tabstop>min_te_production_time</tabstop>
  <tabstop>max_te_production_time</tabstop>
  <tabstop>min_mo_radioactivity</tabstop>
  <tabstop>max_mo_radioactivity</tabstop>
  <tabstop>min_mo_production_time</tabstop>
  <tabstop>max_mo_production_time</tabstop>
  <tabstop>min_name_length</tabstop>
  <tabstop>min_height</tabstop>
  <tabstop>min_weight</tabstop>
  <tabstop>submitButton</tabstop>
  <tabstop>defaultButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
